packages/pass-style/src/types.js

```diff
@@ -87,26 +125,24 @@ export {};
  */
 
 /**
- * @template {Passable} [T=Passable]
+ * @template {Passable} [T=object]
```

Comment:
`object` isn't Passable, is it? I thought that type allows mixing methods and non-function properties. Same applies to later `@template {Passable} [T=object]` patterns as well.

Reply:
Documented now,
```
// TODO a better default type. Ideally it's Passable, but that causes a circular definition.
```